src/components/Sum.jsx

```diff
@@ -1 +1,19 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({
+  a = 0, b = 0,
+}) => (
+  <p>
+    Sum of
+    {' '}
+    {a}
+    {' '}
+    and
+    {' '}
+    {b}
+    {' '}
+    is
+    {' '}
+    {a + b}
```

Comment:
you can use template literal
```suggestion
    {`Sum of ${a} and ${b} is ${a+b}`}
```